src/niscope/system_tests/test_system_niscope.py

```diff
@@ -228,6 +243,47 @@ def test_fetch_measurement(session):
         assert meas == 0.0
 
 
+def test_fetch_measurement_stats(session):
+    test_voltage = 1.0
+    test_record_length = 1000
+    test_channels = range(2)
+    test_num_channels = 2
+    test_num_records = 3
+    session.configure_vertical(test_voltage, niscope.VerticalCoupling.AC)
+    session.configure_horizontal_timing(50000000, test_record_length, 50.0, test_num_records, True)
+    with session.initiate():
+        measurement_stats = session.channels[test_channels].fetch_measurement_stats(niscope.enums.ScalarMeasurement.NO_MEASUREMENT, 5.0)
+
+    assert len(measurement_stats) == test_num_channels * test_num_records
+    for stat in measurement_stats:
+        assert stat.result == 0.0
+
+
+def test_clear_waveform_measurement_stats(session):
```

Comment:
Some of these tests are for functions not added as part of the PR. These should already be in master but if they aren't... could you make a PR to add these so we can push it in ASAP ahead of the two fancy functions in this PR?

Reply:
The only test for which this should be true is test_clear_waveform_measurement_stats which, as mentioned in the previous PR, cannot be meaningfully tested without this PR's addition of fetch_measurement_stats -- hence why the test uses fetch_measurement_stats.